roles/ruby_scl/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+- name: 'Install SCL repository'
+  yum:
+    name: centos-release-scl
+    state: present
+
+- name: 'Install Ruby SCL'
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - "{{ ruby_scl_version }}"
+    - "{{ ruby_scl_version }}-ruby-devel"
+
+- name: 'Enable SCL at login'
+  blockinfile:
+    dest: /etc/profile.d/enable-{{ ruby_scl_version }}.sh
+    create: yes
+    block: |
+      #!/bin/bash
+      source /opt/rh/{{ ruby_scl_version }}/enable
```

Comment:
The modern way is ```source scl_source enable {{ ruby-scl_version }}``` which also sets ```X_SCLS``` for you

Reply:
Good catch!